tools/experiments/experiments.js

```diff
@@ -277,6 +277,12 @@ const EXPERIMENTS = [
     cleanupIssue: 'https://github.com/ampproject/amphtml/issues/8239',
     spec: 'https://github.com/ampproject/amphtml/issues/6829',
   },
+  {
```

Comment:
Ah!  Maybe this is a place where you can leave a comment noting that some of the cleanup has to include the whitelists?